docs/Ajax/ReloadHelper.md

```diff
@@ -0,0 +1,39 @@
+# Haste Ajax\ReloadHelper
+
+This utility class is designed to help handle the ajax reload of frontend modules and content elements.  
+
+
+## Usage
+
+First of all you need to subscribe your frontend module or content element to the events: 
+
+```php
+// Content element
+\Haste\Ajax\ReloadHelper::subscribe(\Haste\Ajax\ReloadHelper::TYPE_CONTENT_ELEMENT, $this->id, ['foo-event', 'bar-event']);
```

Comment:
In the example, I would suggest to subscribe content to `foo-event` and module to `bar-event`. The current example with two events and firing both would never make sense…